pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -1,27 +1,43 @@
-{ lib, fetchFromGitHub, makeWrapper
-, glibcLocales, gobject-introspection, gtk3, libsoup, libsecret
-, buildPythonPackage, python
-, pygobject3, freezegun, mock
+{ lib
+, fetchFromGitHub
+, makeWrapper
+, python
+, buildPythonPackage
+, atk
+, gdk-pixbuf
+, glibcLocales
+, glib-networking
+, gobject-introspection
+, gtk3
+, harfbuzz
+, libsecret
+, libsoup
+, pango
 }:
-
 buildPythonPackage rec {
```

Comment:
Should not this be `buildPythonApplication`?

Reply:
No, for python-packages that is correct

Reply:
This is not a Python package as far as I can tell, though. It might have been missed when we moved apps out of `python-modules/` directory.